src/Sylius/Behat/Context/Ui/PaypalContext.php

```diff
@@ -106,39 +99,19 @@ public function iSignInToPaypalAndPaySuccessfully()
     }
 
     /**
-     * @When I cancel my PayPal payment
+     * @Given /^I(?: have|) cancel(?:ed|) (?:|my )PayPal payment$/
```

Comment:
Same as above. IMHO, it would be much more readable if we would add two different annotations, with different tags.